flask_restalchemy/resources/utils.py

```diff
@@ -80,19 +80,27 @@ def query_from_request(model, model_serializer, request):
     It also paginate the response if a 'page' value is present in the query parameters. A 'per_page'
     value in the query parameters defines the page length, default to 20 items.
 
-    Ordered search is available using 'order_by=<col_name>' providing the name of the model column to be the
-    ordination key. A 'desc' in the query parameters make the ordination in a descending order.
+    Ordered search is available using 'order_by=<col_name>' providing the name list of the model column to be the
```

Comment:
Would be better:
_Ordered search is available using 'order_by=<col_name>'. The minus sign ("-<col_name>") could be used to set descending order_